src/sample/Constants.java

```diff
@@ -7,4 +7,6 @@
 
     // colors
     static String LIGHT_BLUE = "-fx-background-color:#00BFFF; -fx-opacity:1;";
+
+    static  boolean GAME_COMPLETE = false;
```

Comment:
Looks like 2 spaces

Reply:
wdym?

Reply:
Between static and boolean, I think there's 2 spaces

Reply:
was separating those intentionally since they are different things but I am resolving